score-e2e/src/test/java/org/oagi/score/e2e/TS_29_BIEUplifting/TC_29_1_BIEUplifting.java

```diff
@@ -0,0 +1,2693 @@
+package org.oagi.score.e2e.TS_29_BIEUplifting;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.oagi.score.e2e.BaseTest;
+import org.oagi.score.e2e.menu.BIEMenu;
+import org.oagi.score.e2e.obj.*;
+import org.oagi.score.e2e.page.HomePage;
+import org.oagi.score.e2e.page.bie.*;
+import org.oagi.score.e2e.page.code_list.EditCodeListPage;
+import org.oagi.score.e2e.page.code_list.UpliftCodeListPage;
+import org.oagi.score.e2e.page.code_list.ViewEditCodeListPage;
+import org.oagi.score.e2e.page.core_component.ACCExtensionViewEditPage;
+import org.oagi.score.e2e.page.core_component.SelectAssociationDialog;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.math.BigInteger;
+import java.time.Duration;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.oagi.score.e2e.AssertionHelper.*;
+import static org.oagi.score.e2e.impl.PageHelper.*;
+
+@Execution(ExecutionMode.SAME_THREAD)
+public class TC_29_1_BIEUplifting extends BaseTest {
+    private final List<AppUserObject> randomAccounts = new ArrayList<>();
+    String prev_release = "10.9";
+    String curr_release = "10.9.2";
+    AppUserObject usera, userb, developer;
```

Comment:
Please wrap these global properties into a container class to work individually. See TestTC_19_PreConditions in TC_19_1_ReleaseManagement.java.